docs/guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-a-filesystem.md

```diff
@@ -1,5 +1,58 @@
 ---
 title: How to host and share Data Docs on a filesystem
 ---
+import Prerequisites from '../../connecting_to_your_data/components/prerequisites.jsx'
 
-This article is a stub.
+This guide will explain how to host and share Data Docs on a filesystem.
+
+<Prerequisites>
+
+- [Set up a working deployment of Great Expectations.](../../../tutorials/getting-started/intro.md)
+
+</Prerequisites>
+
+Steps
+-----
+
+1. **Filesystem-hosted Data Docs are configured by default for Great Expectations deployments created using great_expectations init.**
+
+  To create additional Data Docs sites, you may re-use the default Data Docs configuration below. You may replace ``local_site`` with your own site name, or leave the default.
+
+    ```yaml
```

Comment:
Same question as in another PR: do we want to un-indent "```yaml" maximally?

Reply:
Resolving comment per discussion earlier today about codeblock alignment.